tests/integration/examples/test_notebooks_gpu.py

```diff
@@ -185,6 +205,23 @@ def test_xdeepfm_integration(
         )
 
 
+@pytest.mark.gpu
+@pytest.mark.integration
+def test_xlearn_fm_integration(notebooks, output_notebook, kernel_name):
```

Comment:
Xlearn does not use GPU, does it?

Reply:
the data in this notebook is pulled using deeprec_utils which imports tensorflow, instead of refactoring all that I just required this to run under gpu, I'll put a todo here to clean this up later